woo-razorpay.php

```diff
@@ -233,8 +233,6 @@ public function __construct($hooks = true)
 
         protected function initHooks()
         {
-            add_action('init', array(&$this, 'check_razorpay_response'));
```

Comment:
@enishant Can we test these changes with VAS merchant keys?

Reply:
Tested standard flow and VAS checkout with latest WordPress + WooCommerce version,  payment flow and callback redirection working fine.

While executing vas checkout, JS script was not loading in document. For testing changed script inclusion as follow to include JS script after DOM loaded within document.

`wp_enqueue_script('razorpay_wc_script');`
**Changed to following**
`wp_enqueue_script('razorpay_wc_script',false, [], false, true );`

@ChetanGN @abdulwahidsharief 
 Please check if we need to include this change as script is executing before DOM loaded.